### PR TITLE
Export normal role name rather than normalized

### PR DESCRIPTION
**Pull request by deanmarcussen**
Fixes https://github.com/OrchardCMS/OrchardCore/issues/5371

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Roles/Recipes/RolesStep.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public async Task ExecuteAsync(RecipeExecutionContext context)
                 if (string.IsNullOrWhiteSpace(importedRole.Name))
                     continue;
 
-                var role = (Role)await _roleManager.FindByNameAsync(importedRole.Name);
+                var role = (Role)await _roleManager.FindByNameAsync(_roleManager.NormalizeKey(importedRole.Name));
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Roles/Recipes/RolesStep.cs]: I think the RoleManager implementation is supposed to do the normalization by itself.

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Roles/Recipes/RolesStep.cs]: https://github.com/dotnet/aspnetcore/blob/master/src/Identity/Extensions.Core/src/RoleManager.cs#L326

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Roles/Recipes/RolesStep.cs]:
```suggestion
                var role = (Role)await _roleManager.FindByNameAsync(importedRole.Name);
```

Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.Roles/Recipes/RolesStep.cs]: Yes else you could fake a role by using different cases on the rolename.

**Pull request merged**